core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -1020,6 +1069,8 @@
                 bkSessionManager.redo();
               });
               return false;
+            } else if (e.which === 123){ // F12
+              bkUtils.Electron.toggleDevTools();
```

Comment:
looks like this is the same from above ^ any way we can register for this in one place?